Docs/Published/API-related/Chunking_API_Documentation.md

```
@@ -0,0 +1,369 @@
+                        text: {type: string}
+                        metadata: {type: object, additionalProperties: true}
+                  original_file_name: {type: string}
+                  applied_options: {type: object, additionalProperties: true}
+```
+
+### 3) GET /capabilities
+
+Discover runtime methods and defaults.
+
+Example response (trimmed):
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)